docs: add user guide on valuta (#2316)
rocky-linux/documentation: docs/desktop/gnome/valuta.md

- Rocky Linux
- Flatpak
- FlatHub

## Installation Process

Go to [Flathub.org](https://flathub.org), type "Valuta" in the search bar and click on **Install**
![Screenshot of the Valuta page on the Flathub website. There, the blue install button is surrounded by a red square](/images/01_valuta.png)

Then, use the following command to install the app in your terminal
`flatpak run io.github.idevecore.Valuta`

## How to Use

To use Valuta, do the following:

1. Choose the country that you currently live in from the dropdown menu and type the amount of cash you want to spend.
![Screenshot of a gray arrow with the number one on top, pointing down in the first section. It shows a grey textbox that has the number, 1,000 written in white with the acronym "USD" next to it](images/02_valuta.png)

1. Choose the country you are traveling to from the dropdown menu.  From there, the converted amount automatically appears.
![Screenshot of a gray arrow with the number two on top, pointing upwards the second section. It shows a green textbox that has the number, 899.52 written in white with the acronym "EUR" next to it](images/03_valuta.png)

## Conclusion

> Whether it’s for a vacation or a business trip, Valuta is the tool that can ease the money conversion process. Eager to learn more about or have more ideas for this application? [Submit an issue in Valuta's repository at GitHub](https://github.com/ideveCore/valuta/issues)

Author: ```suggestion
Whether it’s for a vacation or a business trip, Valuta is the tool that can ease the money conversion process. Eager to learn more about or have more ideas for this application? [Submit an issue in Valuta's repository at GitHub](https://github.com/ideveCore/valuta/issues).
```

﻿---
 title: Valuta
 author: Christine Belzie
 contributors:
---

## Introduction

>  If you travel frequently for businesses or moving to a new country, check out Valuta. It is a tool that quickly converts currencies from two countries.

Author: ```suggestion
 If you often travel or are moving abroad, ease your planning with Valuta. This app quickly converts currencies between two countries.
```

## Assumptions

This guide assumes you have the following:

- Rocky Linux
- Flatpak
- FlatHub

## Installation Process

1. Go to [Flathub.org](https://flathub.org), type "Valuta" in the search bar and click on **Install**
![Screenshot of the Valuta page on Flathub with the blue install button highlighted in a red square](images/01_valuta.png)
2. Copy the manual script in your terminal:

```bash
flapak install flathub io.github.idevecore.Valuta
```

3. Finally, copy the run command and run it in your terminal:

```bash
flatpak run io.github.idevecore.Valuta
```

## How to Use

To use Valuta, do the following:

1. Choose the country that you currently live in from the dropdown menu and type the amount of cash you want to spend.

![Screenshot of Valuta app showing 1000 USD in the input field, with a grey arrow pointing down to a grey box showing 1000 USD](images/02_valuta.png)

2. Choose the country you are traveling to from the dropdown menu.  From there, the converted amount automatically appears.
![Screenshot showing a grey arrow pointing upward to a green box displaying the converted amount, 899.52 EUR](images/03_valuta.png)

## Conclusion

> Whether it’s for a vacation or a business trip, Valuta is the tool that can ease the money conversion process. Eager to learn more about or have more ideas for this application? [Submit an issue in Valuta's repository at GitHub](https://github.com/ideveCore/valuta/issues).

Author: ```suggestion
Whether it’s for a vacation or a business trip, Valuta simplifies currency conversion. Want to learn more or share ideas to improve it? [Submit an issue in Valuta's repository at GitHub](https://github.com/ideveCore/valuta/issues).
```

﻿---
 title: Valuta
 author: Christine Belzie
 contributors:
---

## Introduction

 If you often travel or are moving abroad, ease your finanical planning with Valuta. This application quickly converts currencies between two countries.

## Assumptions

This guide assumes you have the following:

- Rocky Linux
- Flatpak
- FlatHub

## Installation Process

1. Go to [Flathub.org](https://flathub.org), type "Valuta" in the search bar and click  **Install** ![Screenshot of the Valuta page on Flathub with the blue install button highlighted in a red square](images/01_valuta.png)
 
  ![manual install script and run script](images/valuta-install.png)

> 

Reviewer: This is correct! :-)

> 2. Copy the manual script in your terminal:
> 
> ```bash
> flapak install flathub io.github.idevecore.Valuta
> ```

Author: ```suggestion

2. Copy the manual script in your terminal:

    ```bash
    flatpak install flathub io.github.idevecore.Valuta
    ```

```

> 3. Finally, copy the run command and run it in your terminal:
> 
> ```bash
> flatpak run io.github.idevecore.Valuta
> ```

Author: ```suggestion

3. Finally, the manual script in your terminal:

    ```bash
    flatpak run flathub io.github.idevecore.Valuta
    ```

```

## How to Use

To use Valuta, do the following:

1. Choose the country that you currently live in from the dropdown menu and type the amount of cash you want to spend.

![Screenshot of Valuta app showing 1000 USD in the input field, with a grey arrow pointing down to a grey box showing 1000 USD](images/02_valuta.png)

> 

Reviewer: Since the screenshot comes after the first numbered list item, in order for the list to continue with item 2, rather than repeating item 1, it needs to be indented the 4 spaces. Also, while you are correct that the final screenshot (after number 2. here) can be left-justified, leave a blank line between it and the numbered list item.
```
1. Choose the country that you currently live in from the drop down menu and type the amount of cash you want to spend.

    ![Screenshot of Valuta app showing 1000 USD in the input field, with a grey arrow pointing down to a grey box showing 1000 USD](images/02_valuta.png) 

2. Choose the country you are traveling to....

![Screenshot of Valuta app showing 1000 USD in the input field, with a grey arrow pointing down to a grey box showing 1000 USD](images/02_valuta.png)

Author: So...I stick with one screenshot for both steps @sspencerwire?

Reviewer: > So...I stick with one screenshot for both steps @sspencerwire?

No, my bad on that one. I just wasn't paying attention to the screenshot and had the other one in my clipboard, I was concerned just with the positioning. Last screenshot there should have been the one you have for that option. Sorry about that!

﻿---
 title: Valuta
 author: Christine Belzie
 contributors:
---

## Introduction

 If you often travel or are moving abroad, ease your finanical planning with Valuta. This application quickly converts currencies between two countries.

## Assumptions

This guide assumes you have the following:

- Rocky Linux
- Flatpak
- FlatHub

> ## Installation Process
> 
> 1. Go to [Flathub.org](https://flathub.org), type "Valuta" in the search bar and click  **Install** ![Screenshot of the Valuta page on Flathub with the blue install button highlighted in a red square](images/01_valuta.png)

Author: ```suggestion

## Installation process

![Screenshot of the Valuta page on Flathub with the blue install button highlighted in a red square](images/01_valuta.png)

1. Go to [Flathub.org](https://flathub.org), type "Valuta" in the search bar and click  **Install**

```

>  

Reviewer: I think I'd move the screenshot attached to 1. (above) to it's own line above the numbered list item. Like this:
```
## Installation process

![Screenshot of the Valuta page on Flathub with the blue install button highlighted in a red square](images/01_valuta.png)

1. Go to [Flathub.org](https://flathub.org), type "Valuta" in the search bar and click  **Install**
```
That way you are consistent with the image above the option. (I think that's the way the  rest of them are anyway)

 author: Christine Belzie
 contributors:
---

## Introduction

 If you often travel or are moving abroad, ease your finanical planning with Valuta. This application quickly converts currencies between two countries.

## Assumptions

This guide assumes you have the following:

- Rocky Linux
- Flatpak
- FlatHub

## Installation process

![Screenshot of the Valuta page on Flathub with the blue install button highlighted in a red square](images/01_valuta.png)

1. Go to [Flathub.org](https://flathub.org), type "Valuta" in the search bar and click  **Install**
 
  ![manual install script and run script](images/valuta-install.png)

2. Copy the manual script in your terminal:

    ```bash
    flatpak install flathub io.github.idevecore.Valuta
    ```

3. Finally, the manual script in your terminal:

    ```bash
    flatpak run flathub io.github.idevecore.Valuta
    ```

## How to Use

To use Valuta, do the following:

1. Choose the country that you currently live in from the dropdown menu and type the amount of cash you want to spend.

![Screenshot of Valuta app showing 1000 USD in the input field, with a grey arrow pointing down to a grey box showing 1000 USD](images/02_valuta.png)

> 

Reviewer: Still need to indent this screenshot to keep the numbering from starting over at 1. (the screenshot beneath option 1). Still need a blank line after option 2. before the screenshot.
